### ISSUE TITLE
Subtle visual cue for maintaining proportion using color gradients

### ISSUE THREAD

**Issue by BLFiedler** [phetsims/ratio-and-proportion#42]
From @jbphet in the design doc:

> I noticed that the interviewee would have to move things around a bunch whenever he was trying to maintain the ratios while moving and “lost it” in an effort to determine if the ratio was too big or too small.  Should we consider some sort of indicator - and it can be subtle - that indicates whether they are in a “ratio too big” versus “ratio too small” state?  It could be something like a cream colored background versus pure white, or maybe a slight top-to-bottom gradient to the background that switches direction based on the relationship state.

Opening this up for discussion across the team.

I think design considerations are:
- Does the gradient only appear when both cursors are in motion (or in focus)?
- Intensity of the gradient.
- Directionality of the gradient based on movement.

I think modal considerations are:
- Sonification - @jbphet  mentioned he had an idea for how the sound design would follow this visual change.
- Haptic feedback may follow with the sound changes (though we have not had much discussion here yet)
- Description may need to provide this feedback (but perhaps not if it is just a hint and the sound design does a good job of supporting it) - would this be too much to pile on top of existing dynamic responses?

I think pedagogical/interaction considerations are:
- Will the gradient cue them to stay in proportion while moving or cue them to stop and return to the successful proportion they achieved?

**Comment by BLFiedler** [comment]
I think we should visit this topic next we talk about the coloring in the visual display.

**Comment by emily-phet** [comment]
We discussed this in detail in last Tuesday's design meeting. 

Multiple concerns were brought up, including:
- not wanting to overwhelm the existing visual cueing for a more subtle detail
- not wanting to use color alone for visual cueing
- potential issues with introducing new colors, what happens in the transition between colors? In the original bar chart version, colors between pink and green were brown and not desirable. Would probably need to have green only at success point, and one new color for above ratio, and one new color below ratio - concern that this may be too big a change for this stage of the design and too much emphasis on a more subtle detail.
- Concern that we'll run into the same issues for deciding on what color to show as we have run into regarding tolerance - so not sure we can get smooth logical behavior for the cueing.
- The original idea was triggered when observing a person in an interview having a little difficulty maintaining the simultaneous movement task. This original issue might be better addressed by improved tolerance parameters.

Closing this issue for now - will reopen if we see issues in interviews that could be addressed with color cueing.
